tools/eeprom.cc

```diff
@@ -153,6 +204,24 @@ int main(int argc, char* argv[])
         eeprom_file.read((char*)buffer.data(), size);
         eeprom_file.close();
 
+        if (addressingScheme2Bytes == false)
+        {
+            // max eeprom size is 16Kb
+            if ((buffer.size() * 2) > 2_KiB)
```

Comment:
Clarify comment: to go from 16Kb to 2_KiB, I guess it's because we write 32 bits words. But why buffer.size() * 2 ?

Reply:
buffer.size() seems to be the number of works and the `* 2` converts words to byte. (Hopefully i am correct in saying this)